dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/Info.java

```diff
@@ -61,15 +62,35 @@ public Response getTermsOfUse() {
     @GET
     @Path("activeLicenses")
     public Response getActiveLicenses() {
-        List<ActiveLicenseDto> activeLicenses = Lists.newArrayList();
-        licenseRepository.findActiveOrderedByPosition()
-                .forEach(license -> activeLicenses.add(new ActiveLicenseDto(license.getName())));
-
-        return allowCors(response(req -> ok(
-                activeLicenses
-                        .stream()
-                        .map(ActiveLicenseDto::toString)
-                        .collect(Collectors.joining(", "))
-        )));
+        List<ActiveLicenseDto> activeLicenses = licenseRepository.findActiveOrderedByPosition().stream()
+                .map(License::getName)
+                .map(ActiveLicenseDto::new)
+                .collect(Collectors.toList());
+
+        return allowCors(response(r -> ok(new ActiveLicensesResponse(STATUS_OK, activeLicenses))));
```

Comment:
Something is wrong with this `ok()` method. You shouldn't have to define extra class just to add `status` and `data` envelope. IMO we should fix this.